src/game/combat/action.cpp

```diff
@@ -194,8 +194,8 @@ ActionResult PhysicalAttack::calc_score(Combatant& self, Combatant& target) cons
 }
 
 void PhysicalAttack::do_act(Combatant& self, Combatant& target) const {
-    double parry = parry_chance(self, target) + rng().in_range(-5, 5) / 100.0;
-    double evade = evade_chance(self, target) + rng().in_range(-5, 5) / 100.0;
+    double parry = parry_chance(self, target) + rng().flat(-5, 5) / 100.0;
```

Comment:
I think the parry and evade jitter should be symmetric around the "true" value.

Reply:
yes, probably